hazelcast/src/main/java/com/hazelcast/map/impl/query/MapQueryEngineImpl.java

```diff
@@ -155,7 +156,7 @@ private void dispatchQueryOnQueryThreads(Query query, Target target, Collection<
             if (t.getCause() instanceof QueryResultSizeExceededException) {
                 throw rethrow(t);
             }
-            logger.warning("Could not get results", t);
+            logger.log(Level.FINE, "Could not get results", t);
```

Comment:
reduced to keep it consistent with client task. exception here is a not a big deal as it will be retried per-partition. 
it's actually expected when members are bouncing.